packages/react-app-scaffolder/app/index.js

```diff
@@ -10,10 +10,9 @@ const yosay = require('yosay')
 module.exports = class extends Generator {
   _installAndExport() {
     return new Promise(async (resolve, reject) => {
-
       this.log(yosay('Installing dependencies... this may take a minute!'))
 
-      await exec(`yarn prettier --write ./package.json`)
```

Comment:
I know this is old code, but we have `execSync`.  `const exec = promisify(require('child_process').exec)` and `await exec(...)` looks weird to me

Reply:
I see your point, but all the code below is async, so we're better to leave it like that
https://github.com/reapit/foundations/blob/master/packages/react-app-scaffolder/app/index.js#L84
I think all the async tasks will be executed in parallel and it'll not block the main thread while the code is async